admin/schema.php

```diff
@@ -884,6 +884,27 @@
 
 # Release marker: 1.3.0
 
+# Count / occurrences / votes 
+$g_upgrade[210] = array( 'AddColumnSQL', array( db_get_table( 'bug' ), "
+	votes					I		NOTNULL DEFAULT '1'",
```

Comment:
This counter is supposed to indicate how much this bug happenned. So, it does not really make much sense when automatic report would have this at 0.

Besides, even for manual reports "0 votes" seems a bit counter-intuitive to me. Does the feedback reporter himself not want this issue to be implemented? Does the bug reporter not want his bug to be fixed? 

And about history inconsistence - it HAS to be that way. Automatic reporting absolutely should not be reflected in the history. 

The problem here is that unlike manual reporting, auto reports happens at large scale (at least with non-web applications). There can be hundreds, thousands, sometimes million machines and devices with your app installed. Each device can report the same bug, which means you can have millions upvotes. 

Do you really want you bug view page to have a list of a million upvotes from the same reporter account? This would be just a noise, with no useful information.